scripts/preBuildHTML.py

```diff
@@ -1,4 +1,17 @@
-import os
+from subprocess import call
+import os.path
+
 
 def preBuildHTMLFun():
-    os.system("npm run build")
\ No newline at end of file
+    webpackInst = os.path.isdir("../node_modules/webpack")
+
+    if webpackInst == False:
+        print("Running npm install...")
+        call(["npm", "install"])

```

Comment:
I would replace this with `npm ci` instead of `npm install`. This is what I also recommend in the documentation and is the only way to ensure 100% equal packages which is better from a configuration management point of view